src/overview/container.js

```diff
@@ -284,6 +368,18 @@ const mapDispatchToProps = dispatch => ({
     },
     addTag: resultIndex => tag => dispatch(actions.addTag(tag, resultIndex)),
     delTag: resultIndex => tag => dispatch(actions.delTag(tag, resultIndex)),
+    handleFilterPillClick: (tag, option) => event => {
+        event.preventDefault()
+        if (option === 'tag') {
+            dispatch(actions.delTagFilter(tag))
+        } else {
+            dispatch(actions.delDomainFilter(tag))
+        }
+    },
+    handleToggleFltClick: option => event => {
```

Comment:
This and the assoc. action shouldn't use the verb `toggle` in their name - usually reserved for boolean states, and won't take any arg. `setFilterPopup` is probably a better action name as it is setting a many value state.